examples/with-redux/pages/other.js

```diff
@@ -1,23 +1,16 @@
 import React from 'react'
-import { Provider } from 'react-redux'
 import { reducer, initStore, startClock } from '../store'
+import wrapper  from 'next-redux-wrapper';
```

Comment:
Name of the function could be any because it's a default export, it my code it does not even have a name :) but yes, makes sense.